website/components/pluginData.tsx

```diff
@@ -246,6 +246,11 @@ export const DESTINATION_PLUGINS: Plugin[] = [
     logo: "/images/logos/plugins/mssql.svg",
     id: "mssql",
   },
+  {
+    name: "MySQL",
+    logo: "/images/logos/plugins/mysql.png",
```

Comment:
See https://www.mysql.com/about/legal/logos.html for choice of logo

Reply:
They support a vector version (albeit in eps format, but we can convert it to svg), which would be nice to use. I'm happy to update it quickly?